app/controlplane/pkg/metrics/prometheus/collector.go

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheuscollector
```

Comment:
I'd start calling the packages the name of the directory, so this should be called `prometheus`. The caller is the one that will figure out the namign clashes

Reply:
Ok, I'll change the name